examples/transactions.c

```diff
@@ -364,6 +367,9 @@ consumer_group_rebalance_cb (rd_kafka_t *rk,
                              void *opaque) {
         int i;
 
+        if (!strcmp(rd_kafka_rebalance_protocol(rk), "COOPERATIVE"))
+            fatal("COOPERATIVE rebalance protocol not supported");
```

Comment:
maybe "This example has not yet been updated to work with incremental rebalancing". I'm worried the proposed wording makes it seem like transactions don't work with incrreb.